open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/nodes/categories/SubjectAreaCategory.java

```diff
@@ -13,14 +13,14 @@
 public interface SubjectAreaCategory {
 
     /**
-     * @return {@link SubjectAreaEntityClient} for client calls(described in {@link SubjectAreaClient})
+     * @return {@link SubjectAreaNodeClient} for client calls(described in {@link SubjectAreaClient})
      * when working with Category objects
      */
-    SubjectAreaEntityClient<Category> category();
+    SubjectAreaNodeClient<Category> category();
 
     /**
-     * @return {@link SubjectAreaEntityClient} for client calls(described in {@link SubjectAreaClient})
+     * @return {@link SubjectAreaNodeClient} for client calls(described in {@link SubjectAreaClient})
      * when working with SubjectAreaDefinition objects
```

Comment:
@immmus in line with the Terms change ; SubjectAreaDefinition objects => SubjectAreaDefinitions